README.md

```diff
@@ -59,31 +54,35 @@ USB | 0x05
 Bluetooth | 0x06
 Stream | 0x07
 
-&nbsp;
-
 Devices linked with the Dynaudio Connect can be set to one of three zones. Their numeric values are:
-&nbsp;
 
 zone colour | zone value
 ----------- | ----------
 Red | 0x01
 Green | 0x02
 Blue | 0x03
 
-
 ### 2.1. Shortcuts
 
 Not all that is specified is necessary. While volume commands have a specific command code for volume up and down, when specifying absolute volume commands (is there another way?), this distinction can be ignored.
 
 The status value supplied with most commands is only partially processed. It is necessary to specify the correct zone. However, the input part of the status value is ignored.
 
+### 2.2. List commands
+
```

Comment:
Great addition, but I do have a couple of questions. Perhaps you can provide a little more detail on this. 

1. I see you've listed the Command Code but not the Command Value. Is that byte missing for this command? If so, is the payload 4 bytes instead of 5?
- Looking back at my raw data, I suspect the command is "0xFF 0x55 0x01 0x1E 0xE1". If so, the command structure is completely different compared to what is discussed earlier in section 2.  In that case, I think this needs to be in a separate section, just to make the distinction clear. Also, since it is not really a control command. It's more like a command to retrieve Device Information.
2. I see that the Command Code for Device Properties is  0x01. However, that's the same code as for Power On. Are you sure this is correct? 
- If my suspicion mentioned above is correct, this isn't really strange...